hermes-api/src/main/java/pl/allegro/tech/hermes/api/MaintainerDescriptor.java

```diff
@@ -0,0 +1,53 @@
+package pl.allegro.tech.hermes.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+public final class MaintainerDescriptor {
```

Comment:
Descriptor is what identifies the maintainer – id in a given source. It's what needs to be persisted in the storage and I needed a bag for such data. Maintainer contains transient details, currently name only, that we get once we know which source to ask using what id.